opentelemetry-sdk/src/opentelemetry/sdk/context/propagation/b3_format.py

```diff
@@ -100,13 +101,22 @@ def extract(cls, get_from_carrier, carrier):
         )
 
     @classmethod
-    def inject(cls, context, set_in_carrier, carrier):
-        sampled = (trace.TraceOptions.SAMPLED & context.trace_options) != 0
+    def inject(
```

Comment:
this interface change will also need to be propagated up to the API layer as well. There are multiple propagators (tracecontext) that implement the previous API, and those will need to be changed as part of this.

Reply:
fixed too :+1: